src/main/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesFetcher.java

```diff
@@ -53,5 +56,29 @@ public void doPostCleanup(BibEntry entry) {
         new FieldFormatterCleanup(StandardField.ABSTRACT, new ReplaceTabsBySpaceFormater()).cleanup(entry);
         new FieldFormatterCleanup(StandardField.ABSTRACT, new RemoveRedundantSpacesFormatter()).cleanup(entry);
         new FieldFormatterCleanup(StandardField.EDITOR, new RemoveDigitsFormatter()).cleanup(entry);
+        // identifier fields is a key-value field
+        // example: "urn:isbn:978-1-4503-5217-8; doi:10.1145/3129790.3129810; ISI:000505046100032; Scopus 2-s2.0-85037741580"
+        // thus, key can contain multiple ":"; sometimes value seaprated by " " instead of ":"
+        UnknownField identifierField = new UnknownField("identifier");
+        entry.getField(identifierField)
+             .stream()
+             .flatMap(value -> Arrays.stream(value.split("; ")))
```

Comment:
Isn't a regex better / more flexible than this manual string splitting?

Reply:
The initial shot is

```java
Pattern pattern = Pattern.compile("\(([a-z]+:)?\([a-z]+\):\([^; ]+)\(; )?\)*");
```

Maybe, some `()` are still wrong. I find that harder than going "manually" throught he AST

I can use JavaCC (or ANTLR) to generate a grammar and parse the AST of CCSB identifier field. This is the recommended computer science way. Maye at a follow-up pull-request.

Reply:
I like the regex ;-)